network-api/networkapi/wagtailpages/models.py

```diff
@@ -45,7 +45,24 @@
 ]
 
 
-class ModularPage(MetadataPageMixin, Page):
+class MetadataParentMixin(MetadataPageMixin):
```

Comment:
What do you mean with "does not work"? Multiple inheritance should work fine in Python. One important note is that `Meta` classes can clash, so the order of inhertance matters, and you might end up still having to implement a Meta class.

Reply:
I mean the way I have it in the pull request is wrong, doesn't do what I thought it did, and I don't know the correct way to accomplish the actual inheritance. I thought this would be straight forward based on what I read and was told chatting with people, but it still seems like I'm missing some fundamentals about how this all works.